backend/src/handlers/alert.py

```diff
@@ -21,52 +21,36 @@ class AlertModel(BaseModel):
 
 
 @router.get("/")
-def get_alerts(
-    req: Request, include: Union[List[str], None] = Query(default=None)
-) -> JSONResponse:
+def get_alerts(req: Request) -> JSONResponse:
     with req.app.state.db.session() as session:
         alerts: List[Alert] = session.query(Alert).all()
 
-    resp: Dict[str, List[Dict[str, Union[str, int]]]] = {"alerts": []}
-    for alert in alerts:
-        filtered_alert: Dict[str, Union[str, int]] = {}
+    alerts_json: List[Dict[str, Union[str, int]]] = [
+        {
+            "text": alert.text,
+            "startDateTime": str(alert.start_datetime),
+            "endDateTime": str(alert.end_datetime),
+        }
+        for alert in alerts
```

Comment:
Shorter code doesn't always mean better code 🙃

Reply:
How is this bad code exactly?

Reply:
It's not bad, it's just not good either in my opinion. It's not immediately evident that it's iterating over multiple alerts with that inline list comprehension